.github/workflows/notebook_integration_tests.yaml

```diff
@@ -1,4 +1,4 @@
-name: SDK integration tests
+name: Tests
```

Comment:
```suggestion
name: Tests
```

Why not "Notebook integration tests"?

Reply:
It could be. If we have more Notebooks down the line. I'll make that change.